pipeline/train/train.sh

```diff
@@ -129,7 +139,7 @@ opustrainer-train \
     --keep-best \
     --tsv \
     --seed ${seed} \
-    "${extra_params[@]}"
+    "${extra_params[@]}" 2>&1 | $PARSER
```

Comment:
question: Do we actually want to combine stderr and stdout? I imagine the parser handles it fine, but I'm not sure if it has downsides for the `tee` case.